petastorm/unischema.py

```diff
@@ -12,8 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""'unischema' is a data structure definition which can be rendered as native schema/data-types objects
-in several different python libraries. Currently supported arepyspark, tensorflow, numpy"""
+"""A ``unischema`` is a data structure definition which can be rendered as native schema/data-types objects
```

Comment:
The changes in this and codec.py show in https://petastorm.readthedocs.io/en/latest/data_schema.html .